net/toxic/Portfile

```diff
@@ -42,6 +42,9 @@ if {${os.platform} eq "darwin" && ${os.major} < 11} {
     patchfiles-append   0005-Remove-hardcoded-osx_video.patch
 }
 
+# https://github.com/TokTok/toxic/pull/358
+patchfiles-append       0006-Fix-static-asserts.patch
```

Comment:
so why is this still needed in addition to requesting a C11-compatible compiler?

Reply:
@reneeotten Because it will not work otherwise for quite a number of systems.

Reply:
UPD. The patch merged into upstream in https://github.com/TokTok/toxic/commit/b508f7db8c2def4d8d951b3fe2d12c249a911eb1